extensions/gitpod-web/src/util/port.spec.ts

```diff
@@ -0,0 +1,265 @@
+import { ExposedPortInfo, OnPortExposedAction, PortAutoExposure, PortsStatus, PortVisibility, TunneledPortInfo } from '@gitpod/supervisor-api-grpc/lib/status_pb';
```

Comment:
Let's sue the same filename format as other vscode tests rename to `port.test.ts` and move it inside a `test` folder